command-line/src/test/java/graphql/linalyzer/cli/test/utils/OutputChecker.java

```diff
@@ -0,0 +1,101 @@
+package graphql.linalyzer.cli.test.utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class OutputChecker {
+    private final List<Line> expectedOutputLines;
+
+    public OutputChecker() {
+        this.expectedOutputLines = new ArrayList<>();
+    }
+
+    public OutputChecker expect(Line... lines) {
+        expectedOutputLines.addAll(Arrays.asList(lines));
+
+        return this;
+    }
+
+    private String ruleLineWithStyle(String location, String severity, String message, String ruleName) {
+        return String.format(
+                "\t\u001B[37m%s\u001B[0m\t\u001B[33m%s\u001B[0m\u001B[1m%s\u001B[0m\u001B[37m%s\u001B[0m",
```

Comment:
Don't know if we want to check the **exact** output here, including colors, styles, whitespaces, etc.